source/simulator/assemblers/newton_stokes.cc

```diff
@@ -109,16 +109,30 @@ namespace aspect
                 for (unsigned int j = 0; j < stokes_dofs_per_cell; ++j)
                   if (scratch.dof_component_indices[i] ==
                       scratch.dof_component_indices[j])
-                    data.local_matrix(i, j) += ((2.0 * eta * (scratch.grads_phi_u[i] * scratch.grads_phi_u[j]))
-                                                + one_over_eta * this->get_pressure_scaling() * this->get_pressure_scaling()
-                                                * (scratch.phi_p[i] * scratch.phi_p[j]))
+                    data.local_matrix(i, j) += (
+                                                 // top left block: approximate J^{uu}
```

Comment:
It might be good the make the distinction between this 'approximation' and the approximation of `J^{uu}` for the full Newton method. Something like: 'top left block: approximate J^{uu} by A^{uu} when using the defect correction Picard.'

Reply:
Well, it's *not* A^uu. The A^uu block uses the symmetric gradient, but this block here uses the regular gradient.

Reply:
You are right, so we are basically approximating A^uu

Reply:
Correct.